src/data/markdown/translated-guides/en/02 Using k6/04 Thresholds.md

```diff
@@ -271,6 +264,23 @@ We have these thresholds:
 - A counter metric that keeps track of the total number of times content returned was **not** OK.
   The success criteria here implies that content can't have been bad more than 99 times.
 
+**⚠️ Bad practice.** Even though it is possible to define a threshold for the same metric more than once like this:
+
+<CodeGroup labels={["threshold-duplicate.js"]} lineNumbers={[true]}>
+
+```javascript
+export let options = {
+  thresholds: {
+    metric_name: [ 'count<100' ],
+    metric_name: [ 'rate<50' ],
+  }
+};
+```
+
+</CodeGroup>
+
+Such definitions must be avoided so that each metric's threshold is defined only once. Otherwise, only one of the threshold definitions will be executed and there is no guarantee which one it would be. Instead, if the given metric needs more than one threshold expression, put all threshold expressions as part of one array as is shown for `RTT` metric in the previous example.
+
```

Comment:
```suggestion
> **⚠️ Common mistake**: Do not specify multiple thresholds for the same metric by repeating the same object key. Because thresholds are defined as a the properties of a JavaScript object, it's not possible to specify multiple ones with the same property name. Only the last one will remain and the rest will be **silently** ignored. You can specify them with an [array for the same key instead](/using-k6/thresholds/#multiple-thresholds-on-a-single-metric).
```